recovery/src/main/java/com/kin/ecosystem/recovery/restore/view/RestoreEnterPasswordFragment.java

```diff
@@ -116,7 +115,12 @@ public void afterTextChanged(Editable editable) {
 				presenter.onPasswordChanged(editable.toString());
 			}
 		}));
-		openKeyboard(password);
+		password.postDelayed(new Runnable() {
```

Comment:
do you know why the delay is needed? maybe an event can be used instead? (like `OnGlobalLayoutListener`)

Reply:
i'll check

Reply:
@yosriz I've tried, its not working.